packages/tpetra/core/src/kokkos_refactor/Tpetra_KokkosRefactor_Details_MultiVectorDistObjectKernels.hpp

```diff
@@ -802,7 +803,7 @@ outOfBounds (const IntegerType x, const IntegerType exclusiveUpperBound)
     template <typename SC>
     KOKKOS_INLINE_FUNCTION
     void operator() (nonatomic_tag, SC& dst, const SC& src) const {
-      if (Kokkos::abs(dst) < Kokkos::abs(src)) dst = src;
+      if (Kokkos::ArithTraits<Scalar>::abs(dst) < Kokkos::ArithTraits<Scalar>::abs(src)) dst = src;
```

Comment:
```suggestion
      if (Kokkos::ArithTraits<SC>::abs(dst) < Kokkos::ArithTraits<SC>::abs(src)) dst = src;
```

Reply:
Ack, thanks for the catch, fixed in ed2d18e1ecdc31bd471ef550e193ed8498f4a9e0